examples/multimodal/Vision_Fine_tuning_on_GPT4o_for_Visual_Question_Answering_on_Medical_Images.ipynb

```diff
@@ -0,0 +1,816 @@
wwXRwdEQJyXUKOFDwHVedFE37eyxvMjvVCNZG/Uh//EafTF5DGoYjh5CYkk5Oflg9lRTm7DbFjOw882Wgjk2TYjvVkEsRehR4jmg4DdOGqTj8HvnSn/V/pyZ1CsIvaZwvQ3fyL+K5rKuKs0C4EtuQeisX4SJy7+cUCd89p+bnCkUvRRNcSzhwzj3cz+jaODs5s0qfrSjyZFNbM01PO+WTWYllrY3pL2sc6Rs1MDL/hd+UH+cMnbj0n2ATYQOVCK+TgSfrYXwWjGjZz0vj1P+FdnFepsg8hvTGmc6zKc5w4Jn3MKYvoydtnGlKscA4F3tXm9SF+fmUTuVOcKYsyrnGRqiNj5oZ7etAiTqZ5qUof2OaEdhG/nfW4888okD2AnB8lncQeIZ72XA+9TtxUrrNgNRY5xm+9xZkw5Pf1MFxEvidRp1tSTlxnCx9OefQkdWLl34klvt42+qqr1ckp5Gv+CiS06uBrZ5Bsf50AOM4CVPAmVrkcrNCDeV52EZ9vM9e3LiF5I7zRSBl4ieWfqqd+IvKP422Zs501ddAoG2MzHMHZ/oFl8s5zRHmGVjoJ4CJnN2ms4ZN0TaaFl2fA+ct8bIfOsiNO4E//YXymC40zzg0w4XYjfwjLOUCfnHCRfA+rcQK6TDOgFCjD2YfxuxdbVlCsMSL2iVl/EApsnzA36q+K5Q4wYtO/lMtPP4ZuuSaXsh3ws/oUWvdrr2L/vwjXwg/41Stbc4wz8uhLo6xm6Gpz9fJ56zD3maazink2e4gcAo4OXAcBJ7zl/eQc43E3RJS2B74lcPhdPZ4k5GuqxqjhsDiO25KccJNEy2FxxEFsUZh7I4fukIwjy4NHIrV4CxRFsDfFC3PU/L/p4mUm1a4QvkjywXmifBzWbLpn4K/pFpniRPXwNnG2QC8KTAakH/g1PwJeFV+Qu7LlLPbEXTrQpuKbjRtaZwXINoa85/k4YUC55Qg2n2Mcw9dPB9p5v9I/yLyM07btwQ/rx4z22fzR9tAtq5kLwtselHIeabATasLX1pNfDqbPvPeyIsOTIdnGFj6mL9U8PWKNXa2t7Z/9g5VmXiPX9PApoqiGe6EHsVNez7jN6Kx3TCcaGfDHvtNKc+Pg7npCRaqbocFAr8jyjh1lujvtPDV6vRyLtYGG3go9F8F51KekMWw0iJl64x15EiLyTZ4djLNUx/4QjqoCVHiRtGTXKGFmqKdbZR12tFO57IAJWYvnHNK55SqsbpFW3RgmumXWzSc+HSirWJ3RiLLOhNdhSlEayoxcLHTT2RSiSPniTYy//DAqXDiRPIssesl95J+ma6BtfHM+HZpdt0xI7fSSwPL4g+vUFlr7ynzzXzkHyOdqennLL30ES7RXa8JBtJBblyiUxuIsOmyToBjmj1BmF96n8cwOzDlnLP8G3oOL3pPRzij8KqHTfNkz31Y4qq8BNFOEZ7LutbGwMQvIMJoH32OlnWpq/+FvgJW6CvkcfZYmAewZNU7MzxF4R7lnvs7SE7+sMTeEDQD40u8NMbvOzc0soVwDMu/MJO9+E0Nocv+WIhWiRPpN7pbWh8T/gBnVYgARh5g5d7+u6itagEg3tgXTtNFt8IL7bmEN3MJLXf+Vmt6B+P88oB+qntvrKLratOTeYl+3meJZwp8Tj9KPyt+xOFHm9qJC82mkGZWVvxLVszZMyVcqKIrzTk8cDKXKzocwTTn7BJP5hKBCD/BJniRpXPXrCuhwOxHRSmc4yPa6f+KJj7Z01aFZx2GZ4FPhqxPop+Hys3W1Sf8tfeVih+pfSX3kx7+3AzdF9syz8XCTT3idyp4rFtRIfj0d1ewZTkbFo1vSgrh9FOrGZwptGkdLdvARCKXVSG4EP48SW2Kcyk62RM8vuhrvsw8O83yp87mlwILZU2XfqrD+Aba6TetuXvWmHdwKnwRT5/a8FdykcdfJxKHJzjPGb2XGP8MKcHfOP2f4HV14RDJkyYcOKdL4TTGvwBSohMcuNCmMNlA/NPyX0R4Sp+/ii81C12qTPNlyp38keej9DV2B6Xwf5R71kJr3D8FaFMY8LnAN2VPnchzBtgLQOORbrzUv2huOn5g/s/BstDCllmK6bO7b2uJUMOfyxYdc2DKF/CjapVGIzX8SdYqtopPmL+8UH2ygTp3exiKetFZ6TxNT9zZI/GSiznZsKxViT+dkym0oico9+IpcSFgVS7n3Jm4ujllvXiLqo/e0SSU8zhnnicNK5mzlVQ4z1mpKEunxGTHqfXsGXqJT+cUPgcpn5zPRfARLlk5iZy5wwNVuUzP0ie+lJC1KiftI3xJv9DqYYKBS0rTOS/TBF+u5XTKOhPDawZz5At45n17/ppGYC8i1PJMgScaG7h0y7kevhdcqk9q4CPBe5MnU/o5FuKUyPLcQc7Iiy43cJFNqsTEw7PLCtxtC7n7P/fIaqABbNXrh2fTPKOdAL6MkS+gEsh3cFFoOtqiCU52/o/AXeEj5lf8X69LTWmjJ7DEMPt4Fr9/RYZWiB+enxOe7YldlBOTx+jvX6JVEYfz5VeIlagi25izaeJEPPkNUVYhUVg0/gD/MKbTq7bjs48jKlFgmAfvuYSH4TfdaKqSEGvwXA7GFGLCRrQs/0AYAWgMD7xHfrl/aqmKZ28rKVT00uRJgF/mjjMwWkU3HeGngdTknuAiddYK3z1LOZc8J36jQvOf4EwcBiRuOj4nnD98ekYLXNLTXPqF/PXpZAOb1sl05p/nczA1IBy/6ZZ5AXEupc/E0rP8Uy6F7WN76hTyKc55saIRBi6EL04TGVljPGuPf9MLcyknWOJA0eUOnFn/H8XWuGVaV0tjexmyqJ63OB7l/qEHl/KZ8Za13JjIBhwHfko+rkX3QNM5RyPur6b8IgOAWQNJNY2ZFVUiC4SRq0LBaVcJdiVakVOQE4dfSnWXmBPh2xd9qbNiWQ21sE0XnYcEbCgQZqcAhEcD0pxHbpynzLfq5zT80p46JyH/XfnknDj+1IDwqYDflA3
```

Comment:
Since the outputs can either be a short answer or a yes/no, would it be helpful here to break out the yes/no questions into a separate set and see what the accuracy is there? We don't need a model grader for that and I'd be curious to see how the exact accuracy changes there.

Reply:
Separated out the closed form questions!